examples/config files - basic/3 connector-ldap.yml

```diff
@@ -22,6 +22,16 @@ password: "LDAP password goes here"
 host: "LDAP host URL goes here.  e.g. ldap://ldap.example.com"
 base_dn: "defines the base DN. e.g. DC=example,DC=com"
 
+#Optional
+credential_manager:
+    #This will pull credential from Windows Credential Manager in Control Panel.
+    #UserName and Password field above can be commented out
+    #value: windows_credential_manager
+    type: windows_credential_manager
+    #service_name is Required for Windows Credential Manger
+    service_name: "Internet or Network Address field in Credential Manager"
+    username: "UserName"
```

Comment:
I don't get why you are having username be here rather than using the username from above.  That username is available to you in the credentials manager.  If you just want to hide the password, that's fine, but then use username from above.  if you want to hide both the username and the password, that's also fine: just have a username_service and a password_service in the credential_manager section, and have the username be the "password" for the first and the password be the "password" for the second.  But don't be in-between like this.